docs/tutorial.ipynb

```diff
@@ -39,7 +39,6 @@
     "context = MLContext()\n",
     "\n",
     "# Some useful dialects\n",
-    "std = Std(context)\n",
```

Comment:
You should probably register `Func` now (`func = Func(context)`).

Reply:
done!